plugin/storage/es/esRollover.py

```diff
@@ -6,43 +6,64 @@
 import os
 import ast
 import logging
+from pathlib import Path
+import requests
+
 
 ARCHIVE_INDEX = 'jaeger-span-archive'
 ROLLBACK_CONDITIONS = '{"max_age": "7d"}'
 UNIT = 'days'
 UNIT_COUNT = 7
+SHARDS = 5
+REPLICAS = 1
 
 def main():
     if len(sys.argv) != 3:
-        print('USAGE: [INDEX_PREFIX=(default "")] [ARCHIVE=(default false)] [CONDITIONS=(default {})] [UNIT=(default {})] [UNIT_COUNT=(default {})] {} ACTION HOSTNAME[:PORT]'.format(ROLLBACK_CONDITIONS, UNIT, UNIT_COUNT, sys.argv[0]))
+        print('USAGE: [INDEX_PREFIX=(default "")] [ARCHIVE=(default false)] [SHARDS=(default {})] [REPLICAS=(default {})] [CONDITIONS=(default {})] [UNIT=(default {})] [UNIT_COUNT=(default {})] {} ACTION HOSTNAME[:PORT]'.format(SHARDS, REPLICAS, ROLLBACK_CONDITIONS, UNIT, UNIT_COUNT, sys.argv[0]))
```

Comment:
Can you update `HOSTNAME[:PORT]` to be URL, as only seems to work when specifying `http://<host>:9200`.